aicsimageio/readers/tiff_glob_reader.py

```diff
@@ -156,6 +162,11 @@ def indexer(x: str) -> pd.Series:
                 return pd.Series(
                     re.findall(r"\d+", Path(x).name), index=series_idx
                 ).astype(int)
+        elif indexer == "micromanager":
+            def indexer (path_to_img):
```

Comment:
This may need to be typed for linting to pass: `def indexer(x: str) -> pd.Series:`